src/routes/boot.ts

```diff
@@ -57,8 +66,13 @@ export type BootUserReferral = Partial<{
   referralOrigin?: string;
 }>;
 
+interface AnonymousUser extends BootUserReferral {
+  id: string;
+  firstVisit: number;
```

Comment:
Why don't we want to return a string here?
We always return a string for dates

Reply:
Right. Let's stick to using strings. Can't remember exactly which string type we were using (so I used the number as it is easy to remember and works cross-platform), found out it was ISO string.